package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "name-request",
-  "version": "3.2.4",
+  "version": "3.2.5",
```

Comment:
I recommend you go to 3.3.0 to denote that something significant happened since 3.2.x.